.travis.yml

```diff
@@ -1,29 +1,26 @@
-sudo: required 
-dist: trusty 
-language: generic 
-compiler:
-  - gcc
-notifications:
-  email:
-    on_success: always
-    on_failure: always
-#    recipients:
-#      - jane@doe
+sudo: required
+dist: trusty
+language: generic
+
+cache:
+  directories:
+    - $HOME/.ccache
+
 env:
+  global:
+    - CCACHE_DIR=$HOME/.ccache
   matrix:
-    - USE_DEB=true  ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - USE_DEB=true  ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - USE_DEB=true  ROS_DISTRO="jade"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - USE_DEB=true  ROS_DISTRO="jade"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - USE_DEB=true  ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - USE_DEB=true  ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - USE_DEB=true ROS_DISTRO="indigo" ROS_REPO=ros NOT_TEST_INSTALL=true
+    - USE_DEB=true ROS_DISTRO="indigo" ROS_REPO=ros-shadow-fixed NOT_TEST_INSTALL=true
+    - USE_DEB=true ROS_DISTRO="kinetic" ROS_REPO=ros NOT_TEST_INSTALL=true
+    - USE_DEB=true ROS_DISTRO="kinetic" ROS_REPO=ros-shadow-fixed NOT_TEST_INSTALL=true
+
 matrix:
   allow_failures:
-    - env: USE_DEB=true  ROS_DISTRO="indigo"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - env: USE_DEB=true  ROS_DISTRO="indigo"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - env: USE_DEB=true  ROS_DISTRO="jade"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - env: USE_DEB=true  ROS_DISTRO="jade"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
+    - env: USE_DEB=true ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=ros
+    - env: USE_DEB=true ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=ros-shadow-fixed
+
 install:
-  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
+  - git clone --depth=1 https://github.com/ros-industrial/industrial_ci.git .ci_config
 script: 
-  - source .ci_config/travis.sh
+- .ci_config/travis.sh
```

Comment:
idention? works though..

Reply:
Not sure what happened there, this was a direct copy-paste. Fixed now.